lib/internal/timers.js

```diff
@@ -160,8 +160,7 @@ function initAsyncResource(resource, type) {
 
 // Timer constructor function.
 // The entire prototype is defined in lib/timers.js
-function Timeout(callback, after, args, isRepeat, isRefed) {
-  after *= 1; // Coalesce to number or NaN
```

Comment:
I think this line is still necessary for things that are not `undefined`.

Reply:
You're right, the way I was testing this was wrong